notes/dimensions/base.sh

```diff
@@ -145,48 +153,63 @@ http POST "$SERVER/$DATASET/Place" $AUTH \
-#|                  _id                  |              _revision               | id |  name   |                       koord                        
-#| --------------------------------------+--------------------------------------+----+---------+----------------------------------------------------
-#|  2074d66e-0dfd-4233-b1ec-199abc994d0c | 2263e065-e813-4ef2-b43c-9699fc4ed212 |  1 | Vilnius | 0101000020E6100000DDEF5014E8574B40A6ED5F59694A3940
+#|                  _id                  | id |   name    |                       koord                        
+#| --------------------------------------+----+-----------+----------------------------------------------------
+#|  2fcc3da3-be88-4715-83c9-a45bcbeeb3c3 |  1 | Lithuania | 
+#|  2074d66e-0dfd-4233-b1ec-199abc994d0c |  2 | Vilnius   | 0101000020E6100000DDEF5014E8574B40A6ED5F59694A3940
 
 
 http GET "$SERVER/$DATASET/Place?format(ascii)"
-#| _id                                   _revision                             id  name     koord
-#| ------------------------------------  ------------------------------------  --  -------  -------------------------
-#| 2074d66e-0dfd-4233-b1ec-199abc994d0c  2263e065-e813-4ef2-b43c-9699fc4ed212  1   Vilnius  POINT (54.68677 25.29067)
+#| _id                                   id  name       koord      
+#| ------------------------------------  --  ---------  -------------------------
+#| 2fcc3da3-be88-4715-83c9-a45bcbeeb3c3  1   Lithuania  ∅
+#| 2074d66e-0dfd-4233-b1ec-199abc994d0c  2   Vilnius    POINT (54.68677 25.29067)
 
 
 http POST "$SERVER/$DATASET/Location" $AUTH \
-    _id="2074d66e-0dfd-4233-b1ec-199abc994d0c" \
+    _id=$VLN \
     id:=42 \
     name="Vilnius" \
     population:=625349
-#| HTTP/1.1 201 Created
+#| HTTP/1.1 500 Internal Server Error
 #| 
 #| {
-#|     "_id": "2074d66e-0dfd-4233-b1ec-199abc994d0c",
-#|     "_revision": "dcd6e164-5189-4291-b61c-b84d5c1b576c",
-#|     "_type": "dimensions/base/Location",
-#|     "id": 42,
-#|     "name": "Vilnius",
-#|     "population": 625349,
-#|     "type": null
+#|     "errors": [
+#|         {
+#|             "code": "NotImplementedFeature",
+#|             "context": {
+#|                 "attribute": "",
+#|                 "component": "spinta.components.Property",
+#|                 "dataset": "dimensions/base",
+#|                 "entity": "",
+#|                 "feature": "Ability to indirectly modify base parameters",
+#|                 "manifest": "default",
+#|                 "model": "dimensions/base/Location",
+#|                 "property": "name",
+#|                 "schema": "10"
+#|             },
+#|             "message": "Ability to indirectly modify base parameters is not implemented yet.",
+#|             "template": "{feature} is not implemented yet.",
+#|             "type": "property"
+#|         }
+#|     ]
 #| }
+# FIXME: This should not be an error, because we do not modify base, we just use
+#        _base.name as identifier, which exists in Place.
```

Comment:
This is not exactly related with this task, bet could you please fix it, because I can't do further testing.

The error probably comes from https://github.com/atviriduomenys/spinta/pull/425

Reply:
Before writing, added a check,, that compares all inherited variables if they were given.